### DISCUSSION TITLE
Markdown not display the formatted data?

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#11072] [Q&A] [answered]
I add the bullet points on markdown but all the point show in a single line, data not showing in a formatted way? means it need to display as is as we insert on admin side. Please advise

on admin side

![image](https://user-images.githubusercontent.com/5202291/151045129-f58723c1-2e36-4ade-a907-44385a1fb0b6.png)

on frontside:

![image](https://user-images.githubusercontent.com/5202291/151044834-dd424ed5-151b-4599-b738-8cfe67c2b40b.png)

**Comment by hishamco** [comment, marked as answer]
Did you render them as Html?

**Reply by hishamco** [reply]
Use `MarkdownToHtmlAsync()` to render them as HTML

**Reply by saleemhaq** [reply]
if we have a css custom class that need to put on each li and ul so how could i achieve?

**Reply by hishamco** [reply]
I think the above answer your question. To apply a certain CSS class you can surround your elements with div and apply the style directly

**Comment by saleemhaq** [comment]
@hishamco Can u also please advise on that item below:
https://github.com/OrchardCMS/OrchardCore/discussions/11055

**Reply by hishamco** [reply]
Hope to find tomorrow and check
